db/model/model.js

```diff
@@ -1,33 +1,44 @@
-const db = require('../connection')
+const db = require("../connection");
 
 exports.fetchTopics = () => {
-    return db.query(`SELECT * FROM topics`).then(({
-        rows }) => {
-            return rows
-        })
-}
+  return db.query(`SELECT * FROM topics`).then(({ rows }) => {
+    return rows;
+  });
+};
 
 exports.fetchArcticlesById = (id) => {
-    return db.query(`SELECT * FROM articles WHERE article_id=$1`, [id]).then(({ rows }) => {
-        if (rows.length === 0) {
-            return Promise.reject({status: 400, msg: 'Id not found'})
-        }
-        return rows
-    })
-}
-
-exports.fetchUsers = () => {
   return db
-    .query(`SELECT * FROM users`)
+    .query(`ALTER TABLE articles ADD comment_count INT NOT NULL DEFAULT 0 `)
```

Comment:
We don't want to alter the table here, we can use a `JOIN` to get a comment count from the combined articles and comments table. There is an example here [SQL - Foreign keys, Joins and Aggregate Functions](https://notes.northcoders.com/courses/js-back-end/sql-relationships)